freiburgermsu/hillfit
# Updates (#8)

* expand fitting.py, add an example Notebook, and create a setup.py file

* compose the RSTREADMEand complete the setup.py file, with the updated example

* compose the RSTREADMEand complete the setup.py file, with the updated example

* updating the RST file, & .gitignore

* Delete examples/.ipynb_checkpoints directory

* Delete dist directory

* Delete Hillfit.egg-info directory

* update the export path terminology

* simplify the example logic

* clarify the equation string

* update the RSTREADME

* update the README

* compose the unit-test and the YAMLAction

* compose the unit-test and the YAMLAction

* update the YAMLAction

* update the YAMLAction

* update the YAMLAction

* update the YAMLAction

* update the YAMLAction

* update the YAMLAction

* update the README

* update the returned params

* update the returned params

* example update

* example update

* move the R^2

* remove redundant returns, and update the README

* update the README

* update the README

* simplify the export

* minor revisions

* self-accessible R-squared

* self-accessible R-squared

* update the unit-test

* update the unit-test

* update the unit-test

* update the unit-test

* minor updates

* [pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

* README

* README

* merge edits

* R^2

Co-authored-by: Andrew Freiburger <Andrewefreiburger@gmail.com>
Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
